### PR TITLE
Docs edit faiss for clarity and readability

### PR DESCRIPTION
**Pull request by educatorsRlearners**
Made several changes in formatting to improve the readability of the FAISS example. Namely,

- removed duplicate table of contents
- fixed grammatical mistakes
- edited for wordiness
- corrected typos

Additionally, I expanded the 'Requirements' section to make it more explicit.

### CODE REVIEW

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: faiss-search/README.md [hunk 1]
```diff
@@ -35,37 +16,58 @@
 
 </p>
 
-In this demo, we use Jina to build a vector search engine that finds the closest vector in the database to a query one. We will use FAISS (https://github.com/facebookresearch/faiss), the Facebool AI Similarity Search, the library for efficient similarity search and clustering of dense vectors. This example uses ANN_SIFT10K dataset (http://corpus-texmex.irisa.fr/). This dataset is formed by 10K vectors to index, 100 vectors to query and 25K vectors to train the vector index. These vectors are SIFT descriptors for some image dataset. The example can very easily be adapted to work with larger datasets from the same source such as ANN_SIFT1M, ANN_GIFT1M or ANN_SIFT1B. For this demo, a vector is considered to be a document and only one chunk per document is used.
+In this demo, we use Jina to build a vector search engine that finds the closest vector in the database to a query. We will use the Facebook AI Similarity Search, ([FAISS](https://github.com/facebookresearch/faiss)), which is a library for efficient similarity search and clustering of dense vectors. This example uses [ANN_SIFT10K](http://corpus-texmex.irisa.fr/), which is a dataset comprised of three vectors:
```
[the remark this answers is not in the document]
Comment (educatorsRlearners) [reply faiss-search/README.md]: I've made the requested changes.

File: faiss-search/README.md [hunk 2]
```diff
@@ -109,8 +119,10 @@ pods:
 </tr>
 </table>
 
-### Query
-As in the indexing time, we also need a Flow to process the request message during querying. The only difference with its index counterpart is that `doc_indexer` is piped to the `faiss_indexer` with a `ranker` in the middle.
+### Query <!-- omit in toc -->
+
+Just as we need to index, we also need a Flow to process the request message during querying. The only difference with its index counterpart is that `doc_indexer` is piped to the `faiss_indexer` with a `ranker` in the middle (see below).
```
[the remark this answers is not in the document]
Comment (educatorsRlearners) [reply faiss-search/README.md]: Done

File: faiss-search/README.md [hunk 3]
```diff
@@ -145,29 +157,29 @@ pods:
 </tr>
 </table>
 
-In this Flow, the `faiss_indexer` is the one that will do the nearest neighbours search from the given chunk (in this case, since every document has one chunk they are the same). Later, the `MinRanker` ranks the chunks by min score value from all the retrieved chunks from `faiss_indexer`, Later, `doc_indexer` retrieves the actual document value from the Document Id.
+In this Flow, the `faiss_indexer` is the one that will do the nearest neighbours search from the given chunk (in this case, since every document has one chunk they are the same). Later, the `MinRanker` ranks the chunks by minimum score value from all the chunks retrieved from the `faiss_indexer`, Later, `doc_indexer` retrieves the actual document value from the Document Id.
```
[the remark this answers is not in the document]
Comment (educatorsRlearners) [reply faiss-search/README.md]: Done

File: faiss-search/README.md [hunk 4]
```diff
@@ -222,22 +233,25 @@ With the default demo, the results are:
...
-- Try different kind of inverted indices and options from FAISS.
-- Try other indexers or rankers.
-- Try indexing larger datasets.
-- Play around with different evaluation metrics.
 
-## Documentation 
+Where to go from here? You can always try:  
+
+- different kinds of inverted indices and options from FAISS.
+- other indexers or rankers.
```
[the remark this answers is not in the document]
Comment (educatorsRlearners) [reply faiss-search/README.md]: Done

**Review by JoanFM — APPROVED** [review]

**Pull request merged**